centreon/www/include/common/form/form.css

```
@@ -0,0 +1,1257 @@
+ * accordion sections, modern inputs, consistent with cl-* listing styles.
+ */
+
+/* ==========================================================================
+   Form container
+   ========================================================================== */
+
+.cf-form-wrapper {
+    max-width: 900px;
+    margin: 0 auto;
+    padding: 16px 20px 0;
+    font-family: 'Roboto', -apple-system, BlinkMacSystemFont, sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Roboto" (font-family-name-quotes)

(font-family-name-quotes)